website/docs/deployment.mdx

```diff
@@ -85,6 +85,16 @@ export default function Home() {
 }
 ```
 
+## Protecting your site {#protecting-your-site}
+
+The following are a couple of potential ways to protect sensitive content on a Docusaurus site:
+
+1. Password protection, some static hosts and CDNs offer this service. For example, [Netlify](#deploying-to-netlify) provides a [password based]( https://docs.netlify.com/visitor-access/password-protection/) function for controlling content access.
+
+2. Cookie or OAuth workflow, CDNs with edge serverless functions support this feature set. For example, using [Cloudflare](#deploying-to-cloudflare-pages) workers an authentication workflow like [GitHub Oauth](https://github.com/gr2m/cloudflare-worker-github-oauth-login) can be implemented.
```

Comment:
technically edge serverless permits anything (including any SSO, Oauth, SAML, htaccess/password protection...) and it's what CDNs use themselves to handle 1). It's a bit like 1 and 2 are the same case somehow. Wonder what's the best way to present that.